genkit-tools/common/src/eval/validate.ts

```diff
@@ -92,9 +92,24 @@ function validate(
   data: unknown
 ): { valid: boolean; errors?: ErrorDetail[] } {
   const isModelAction = actionRef.startsWith('/model');
-  let input = isModelAction
-    ? getModelInput(data, /* modelConfig= */ undefined)
-    : data;
+  let input;
+  if (isModelAction) {
+    try {
+      input = getModelInput(data, /* modelConfig= */ undefined);
+    } catch (e) {
+      return {
+        valid: false,
+        errors: [
+          {
+            path: '(root)',
```

Comment:
Just curious, what is this path like usually? Why are we setting it to (root) here?

Reply:
In zod schema validation, if the error is at a sub field down the heirarchy, this path specifies which field it is, (root) is when the overall object is wrong, which is suitable for this check.